source/regression1.md

```diff
@@ -999,25 +1010,25 @@ sacr_fit = GridSearchCV(
     scoring="neg_root_mean_squared_error"
     ).fit(
       sacramento_train[["sqft", "beds"]],
-      sacramento_train[["price"]]
+      sacramento_train["price"]
     )
 
 # retrieve the CV scores
-sacr_results = pd.DataFrame(sacr_fit.cv_results_)[
-    ["param_kneighborsregressor__n_neighbors", "mean_test_score", "std_test_score"]
-]
-sacr_results = sacr_results.assign(
-    sem_test_score = sacr_results["std_test_score"] / 5**(1/2)
-).rename(
-    columns = {"param_kneighborsregressor__n_neighbors" : "n_neighbors"}
-).drop(
-    columns = ["std_test_score"]
-)
+sacr_results = (
+    pd.DataFrame(sacr_fit.cv_results_)[[
+        "param_kneighborsregressor__n_neighbors",
+        "mean_test_score",
+        "std_test_score"
+    ]]
+    .assign(sem_test_score=sacr_results["std_test_score"] / 5**(1/2))
```

Comment:
@joelostblom this is incorrect for the same reason as above, but this time it will fail silently (it'll use the old `sacr_results` df from before)

I will fix it